pocs/utils/social_slack.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+from pocs.utils.logger import get_root_logger
+
+
+class SocialSlack(object):
+
+    """Messaging class to output to Slack
```

Comment:
Nit: please use punctuation. And in this case I'd end the docstring on that first line, e.g.:

`"""Messaging class to output to Slack."""`